_entities.md

```diff
@@ -455,6 +494,7 @@ memberAt | The date when the user has become a verified member.
 - **otp.transactions.send.enabled** - This will prompt the user to input an OTP token when creating a transaction to another user.
 - **otp.transactions.transfer.enabled** - This will prompt the user to input an OTP token when transacting between the user's own cards.
 - **otp.transactions.withdraw.crypto.enabled** - This will prompt the user to input an OTP token when withdrawing to the bitcoin network.
+- **otp.vmc.enabled** - This will prompt the user to input an OTP token when using VMCs.
```

Comment:
Is "VMC" defined anywhere in the documentation? I couldn't find anything.

Reply:
For future reference, I've discussed this with @rplopes offline, and turns out this field is getting deprecated; so for now it will be added to the docs to match the actual API output, but it's not worth describing it in detail as it will be gone soon anyway.